pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -553,7 +577,7 @@ func buildDefaultCluster(env *model.Environment, name string, discoveryType v2.C
 	}
 
 	defaultTrafficPolicy := buildDefaultTrafficPolicy(env, discoveryType)
-	applyTrafficPolicy(cluster, defaultTrafficPolicy, nil)
+	applyTrafficPolicy(cluster, defaultTrafficPolicy, nil, env)
 	return cluster
```

Comment:
And since you have to make a change anyway (remove the debug), can you please pass the env as the first param instead of the last one ?

Reply:
updated.